services/motion/builtin/builtin_utils_test.go

```diff
@@ -268,3 +269,31 @@ func createMoveOnMapEnvironment(
 	test.That(t, err, test.ShouldBeNil)
 	return kb, ms
 }
+
+func TestCorrectStartPose(t *testing.T) {
```

Comment:
1. Could we please have t.Runs that describe what the inputs mean & what the expected outputs are?
2. Could we please test for errors?

Reply:
Done.